Overview.src.html

```diff
@@ -3979,9 +3990,52 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 is itself associated with <span title=concept-Response-response>response</span>'s
 <span title=concept-response-header-list>header list</span>.
 
-<p>A <code>Response</code> object's <span title=concept-Body-body>body</span> is its
-<span title=concept-Response-response>response</span>'s
-<span title=concept-response-body>body</span>.
+<p>A <code>Response</code> object also has an associated
+<dfn title=concept-Response-readable-stream>readable stream</dfn> which is null or a
+<code title=concept-ReadableStream>ReadableStream</code> object. Its initial value is null.
+
+<p class="XXX">This might become a <code>ReadableByteStream</code> object, depending on the
+<a href="https://github.com/whatwg/streams/issues/379">the discussion</a>. The type might be
```

Comment:
"the the discussion" is wrong. I think this entire thing can be clearer as:

`<a>This might become a ReadableByteStream object</a>. While the type might change, the behavior specified will be equivalent since the hypothetical ReadableByteStream object will have the same members as the ReadableStream object has today.`